c6021light/test/integration/StatelessRouting.cpp

```diff
@@ -227,6 +240,7 @@ TEST_P(PowerRoutingFixture, PowerRequest_FromI2C) {
   // Expoected output packets
   EXPECT_CALL(canTx, SendPacket(canFrame));
   EXPECT_CALL(lnTx, DoAsyncSend(LnPacket));
+  EXPECT_CALL(xnHal, setPower(power ? csNormal : csTrackVoltageOff));
```

Comment:
Not exactly an ideal solution. Can we come up with a better abstraction?

Affects three locations in this file.